forEach.js

```diff
@@ -0,0 +1,16 @@
+let family = ["Sofi","Hen", "Len"];
+
+function forEch(arr, callback){
+  for(let i = 0; i < arr.length; i++){
+    let item = (arr[i]);
+    let index = (i);
```

Comment:
1. Move to constants, no need to change them
2. remove parentheses

```suggestion
    const item = arr[i];
    const index = i;
```